UDV-Core/src/Utils/BaseDemo/js/BaseDemo.js

```diff
@@ -346,33 +346,13 @@ export class BaseDemo {
     }
 
     /**
-     * Initialize the iTowns 3D view.
+     * Adds Lyon WMS layer to the iTowns 3D view.
```

Comment:
Could you be more specific please ? e.g. adds WMS elevation Layer of Lyon in 2012 and WMS imagery layer of Lyon in 2009 (from Grand Lyon data)